wp-content/themes/archivepoliticalads/content-explore_candidates.php

```diff
@@ -1,34 +1,82 @@
 <div class="row">
     <div class="col-md-12">
-        <div id="explore-candidates-title" class="explore-subtitle-row">
-            <h3>Candidates</h3>
-            <p>Browse ads by candidate</p>
+        <div id="explore-candidates-title" class="explore-subtitle-row text-center">
+            <h3>Political Ads by Candidate</h3>
         </div>
     </div>
 </div>
 
 <div class="row">
-    <div class="col-md-12">
-        <div id="explore-candidates-content" class="explore-content-row">
-            <div id="explore-candidates-slider" class="explore-list explore-slider">
-            <?php
-                $candidates = get_candidates();
-                foreach($candidates as $candidate) {
+    <div class="col-md-12 col-lg-8 col-lg-offset-2 ">
+        <div id="explore-candidates-content" class="explore-content-row text-center">
+            <ul class="nav nav-pills" id="candidate-race-pills" role="tablist">
+                <?php
+                    $races = get_races();
+                    $i = 0;
+                    foreach($races as $race) {
                     ?>
-                    <a href="<?php bloginfo('url'); ?>/browse/?q=<?php echo(urlencode("candidate:\"".$candidate['name']."\""));?>">
-                        <div class="explore-item item">
-                            <div class="explore-wrapper">
-                                <div class="explore-label"><?php echo($candidate['name']); ?></div>
-                                <div class="explore-count"><?php echo($candidate['count']); ?>  Ad<?php echo($candidate['count']==1?"":"s");?></div>
-                            </div>
-                            <div class="explore-image">
-                                <img src="<?php echo(get_archive_image_url($candidate['name'])); ?>" />
+                    <li role="presentation" <?php if ($i == 0) echo 'class="active"';?>><a href="#<?php echo($race['name']); ?>" aria-controls="<?php echo($race['name']); ?>" role="tab" data-toggle="pill"><?php echo($race['name']); ?></a></li>
+                <?php $i++; } ?>
```

Comment:
(Nit picky as well) -- if this was just a bracket that'd be one thing, but having logic + bracket I would break this into multiple lines for legibility.